types/request-promise-native/index.d.ts

```diff
@@ -8,8 +8,8 @@ import http = require('http');
 
 declare namespace requestPromise {
     interface RequestPromise extends request.Request {
-        then<TResult>(onfulfilled?: (value: any) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => TResult | PromiseLike<TResult>): Promise<TResult>;
-        then<TResult>(onfulfilled?: (value: any) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => void): Promise<TResult>;
+        then<TResult>(onfulfilled?: (value: request.RequestResponse) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => TResult | PromiseLike<TResult>): Promise<TResult>;
+        then<TResult>(onfulfilled?: (value: request.RequestResponse) => TResult | PromiseLike<TResult>, onrejected?: (reason: any) => void): Promise<TResult>;
```

Comment:
This isn' true. See [`request-promise`](https://www.npmjs.com/package/request-promise) examples; the default is to return the response body, but you could also be returning the full response object.